Production/govuk_ios/Coordinators/SettingsCoordinator.swift

```diff
@@ -6,21 +6,25 @@ class SettingsCoordinator: TabItemCoordinator {
     private let viewControllerBuilder: ViewControllerBuilder
     private let deeplinkStore: DeeplinkDataStore
     private let analyticsService: AnalyticsServiceInterface
+    private let deviceInformation: DeviceInformationInterface
```

Comment:
You don't need to hold on to the deviceInformation, it's not used anywhere in the class.  Nor are analyticService or coordinator builder, if you want to remove those declarations as well.  Keeps it all easier to read and understand.
In fact coordinatorBuilder isn't passed on either so can removed from the initializer as well, if you are feeling ambitious

Reply:
Good spot. I've removed the unused properties